core/modules/views/includes/view.inc

```diff
@@ -49,10 +50,10 @@ class view {
    */
   var $human_name;
 
-  /**
-   * The core version the view was created for.
-   * @var int
-   */
+  /**	
+   * The core version the view was created for.	
+   * @var int	
+   */	
```

Comment:
This seems to be adding whitespace...?

Reply:
Fixed ...but this entire bit will be removed anyway, as part of https://github.com/backdrop/backdrop-issues/issues/2432